src/pages/ProfilePage.js

```diff
@@ -143,6 +136,13 @@ function ProfilePage(props) {
 
     const chatReportWithCurrentUser = !isCurrentUser && !Session.isAnonymousUser() ? ReportUtils.getChatByParticipants([accountID]) : 0;
 
+    // eslint-disable-next-line rulesdir/prefer-early-return
+    useEffect(() => {
+        if (ValidationUtils.isValidAccountRoute(accountID) && !hasMinimumDetails) {
```

Comment:
We should have prevented the fetching if the user was concierge. This caused https://github.com/Expensify/App/issues/61178